unit-tests/live/aus/test-api.py

```diff
@@ -0,0 +1,106 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import devices, log, test, file, repo
+from time import sleep
+
+#############################################################################################
+# Tests
+#############################################################################################
+
+test.start("Test set empty string counter")
+try:
+    rs.aus_set("", 0)

```

Comment:
Here is a good place for a `try` :)
If you take latest development you can use [check_throws](https://github.com/IntelRealSense/librealsense/blob/a3c28d3ba6440fdb6e19b45bcc0e3be27b8ea314/unit-tests/py/rspy/test.py#L319)